File: elasticsearch-connect.go
package elasticsearchConnect

import (
	"fmt"
	"log"

	"gopkg.in/olivere/elastic.v3"
)

func main() {
	client, err := elastic.NewClient(
		// Compose connection strings
		elastic.SetURL("https://username:password@portal113-2.latest-elasticsearch.compose-3.composedb.com:10113/", "https://username:password@portal164-1.latest-elasticsearch.compose-3.composedb.com:10164/"),
		elastic.SetSniff(false),
	)
	if err != nil {
		log.Fatal(err)
	}

	// build the cluster health call
	health, err := client.ClusterHealth().Do()
	if err != nil {
		log.Fatal(err)
	}
	fmt.Printf("<------ Cluster Health ------>\n%+v\n", health)
}

File: elasticsearch-connect.js
const elasticsearch = require('elasticsearch');

const client = new elasticsearch.Client({
// Compose connection strings
  hosts: [
    "https://username:password@portal113-2.latest-elasticsearch.compose-3.composedb.com:10113/", "https://username:password@portal164-1.latest-elasticsearch.compose-3.composedb.com:10164/"]
});

// view the health of a cluster
client.cluster.health((err, res) => {
  if (err) throw err;
  console.log(res);
  client.close();
  process.exit(0);
});
